app/scripts/controllers/preferences-controller.ts

```diff
@@ -127,15 +127,18 @@ export type Preferences = {
 // Omitting showTestNetworks and smartTransactionsOptInStatus, as they already exists here in Preferences type
 export type PreferencesControllerState = Omit<
   PreferencesState,
-  'showTestNetworks' | 'smartTransactionsOptInStatus'
+  | 'showTestNetworks'
+  | 'smartTransactionsOptInStatus'
+  | 'privacyMode'
+  | 'tokenSortConfig'
+  | 'useMultiRpcMigration'
```

Comment:
Omitted as in extension they are used as properties of `preferences` property of the state.